### ISSUE TITLE
Support custom children as layer content for ParallaxBanner

### ISSUE THREAD

**Issue by jscottsmith** [jscottsmith/react-scroll-parallax#22]
Should be possible to allow custom children for each layer of the `<ParallaxBanner>`. This would allow video or other html to parallax inside a banner. See #21

```jsx
const Video = ({ src }) => (
    <video src={src}></video>
);

...

<ParallaxBanner
    layer={[{
        amount: 0.2,
        children: <Video src="foo.mp4">,
    }]}
/>
```

**Comment by webberwang** [comment]
Yeah I can't get text to parallax, only images

**Comment by jscottsmith** [comment]
Ya not quite possible at the moment. This should be an easy thing to add though so I'll try and get some work done on it this week. Hang tight.

**Comment by jscottsmith** [comment]
Published [v1.3.2](https://github.com/jscottsmith/react-scroll-parallax/releases/tag/v1.3.2)
